### PR TITLE
add instructions about how to understand the performance of sync server and scale it

### PR DESCRIPTION
**Pull request by wei-lee**
This PR:

* added a new sync_configuration_guide.adoc and move the `Configuring Frequency of the Sync Server` section to there. Plan to add more there for other configuration options.
* updated the sync performance and scale guide to add more information about how to inspect and understand the performance of the sync server, and how to scale it.

ping @david-martin  for review.

### CODE REVIEW

File: sync_performance_scaling_guide.adoc [hunk 1]
```diff
@@ -1,66 +1,119 @@
...
 
-=== How long it takes the sync processor to run?
+=== 1. Query the `/mbaas/sync/stats` endpoint
 
-The sync frequency value will decide how long the system should wait between sync processor executions.
-The sync frequency is the time from the completion of the one execution to the start time of next execution.
-This means there will never be a situation where 2 sync processors will be running at the same time.
-It also means:
+By default, the sync framework will save its metrics data into Redis while it's running. 
+You can then send a HTTP GET request to the `/mbaas/sync/status` endpoint to view the summary of those metrics data.
```
Comment (david-martin) [inline sync_performance_scaling_guide.adoc]: typo? /mbaas/sync/stats instead of /mbaas/sync/status

File: sync_performance_scaling_guide.adoc [hunk 2]
```diff
@@ -1,66 +1,119 @@
...
-The rate will be fairly consistent, so you can perform load tests against the backend to determine if your backend can handle that much load.
+If you want to visualize the current and historical metrics data, you can instruct the sync server to send the metrics data to InfluxDB, and see the graphs in Grafana.
 
-However, its worth mentioning that this value will not grow when you scale the app.
-For example, if you have multiple workers in your server, the sync processor executions are distributed among the workers rather than duplicated among them.
-This is by design to protect the backend when the app is under heavy load.
+There are plenty of tutorials online to help setup InfluxDB and Grafana. For example:
 
-=== How much extra load it will cause to the server itself?
+* http://www.netinstructions.com/installing-influxdb-and-grafana-on-an-ec2-instance/[How to setup InfluxDB and Grafana on AWS EC2]
```
Comment (david-martin) [inline sync_performance_scaling_guide.adoc]: Might be better to just point to the 1 upstream example for setting up influx & grafana via docker/openshift.

File: sync_performance_scaling_guide.adoc (hunk 1 above)
Comment (wei-lee) [reply sync_performance_scaling_guide.adoc]: yeah, it's a typo

File: sync_performance_scaling_guide.adoc (hunk 2 above)
Comment (wei-lee) [reply sync_performance_scaling_guide.adoc]: ok, we can do that.

**Pull request merged**